### PR TITLE
Fields API

### PR DESCRIPTION
**Pull request by JarJak**
As an addition to #150 I've added API endpoints for fields, so now you can make a request like 
`GET /api/contents/1/fields.json` 
and it will return all names and values fields of this content. Values are always arrays here, like:
```
[
  {
    "name": "Title",
    "value: [
      "Hello world!"
    ]
  }
]
```

TBD: I can also add type of field to it in case someone will find it is useful information in frontend for proper rendering.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
         "behat/mink-extension": "^2.3",
         "behat/mink-goutte-driver": "^1.2",
         "behatch/contexts": "^3.2",
-        "coduo/php-matcher": "^3.1.2",
+        "coduo/php-matcher": "dev-patch-2@dev",
```
Comment (bobdenotter) [inline composer.json]: Not a blocking issue, but is this `coduo/php-matcher` robust? I think this is the second or third time we've had to use it as non-stable, right?

Comment (JarJak) [reply composer.json]: Because I've made another PR to them ;)

**Comment by JarJak** [comment]
>TBD: I can also add type of field to it in case someone will find it is useful information in frontend for proper rendering.

@bobdenotter wdyt? would it be useful for headless frontend?

**Comment by bobdenotter** [comment]
> wdyt? would it be useful for headless frontend?

Yes, very much! it'll be great for context. If something has a value of `foo`, and it comes from a `type: text` you might want to display it differently than something that's a `type: select`. :-)

**Comment by JarJak** [comment]
@bobdenotter type added in API response. Also I've made Dynamic DiscriminatorMap :D

**Pull request merged**